client/src/components/auth/RequireAuth.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import axios from 'axios';
+
+axios.interceptors.request.use(
+  options => {
+    options.headers.authorization = localStorage.getItem('jwt');
+    return options;
+  },
+  error => Promise.reject(error)
```

Comment:
Since you're just passing along the error here, I wonder if you could just use `Promise.reject` for this callback. It depends primarily on if this callback _can_ take more arguments, and, if so, can `Promise.reject` as well?
Try it out if you want:
```javascript
// ...
.use(
  options => {
    options.headers.authorization = localStorage.getItem('jwt');
    return options;
  },
  Promise.reject
);
```